examples/README.md

```diff
@@ -571,6 +572,12 @@ Example | Description
 [Viewport Node](../examples/ui/viewport_node.rs) | Demonstrates how to create a viewport node with picking support
 [Window Fallthrough](../examples/ui/window_fallthrough.rs) | Illustrates how to access `winit::window::Window`'s `hittest` functionality.
 
+## Usage
+
+Example | Description
+--- | ---
+[Context Menu](../examples/usages/context_menu.rs) | Example of a context menu
```

Comment:
For further efforts: This is generated by `cargo run -p build-templated-pages -- build-example-page` as per https://github.com/tygyh/bevy/blob/ebab7d32ca3d68cfe2605c5196edbcdc2b1c87b1/examples/README.md#L2